mrp_unbuild_cust_alu_analytics/models/mrp_unbuild.py

```diff
@@ -27,7 +27,7 @@ class MrpUnbuild(models.Model):
     @api.onchange('product_id')
     def _onchange_product_id(self):
         if self.product_id and self.process_type_id:
-            self.bom_id = self.product_id.product_tmpl_id.bom_ids.filtered(lambda r: r.process_type_id == self.process_type_id)
+            self.bom_id = self.product_id.product_tmpl_id.bom_ids.filtered(lambda r: r.process_type_id == self.process_type_id)[0]
```

Comment:
Origin code didn't fail in one case: when there wasn't any BoM for the selected process type (as filtered returned `False` and, then, simply assign `False`). New code could fail in that case (because `False[0]` will raise an error), please protect it.